continuous_integration/environment-3.10-dev.yaml

```diff
@@ -5,9 +5,9 @@ channels:
 dependencies:
 - c-compiler
 - dask>=2022.3.0
-# FIXME: handling is needed for httpx-based fastapi>=0.87.0
-- fastapi>=0.69.0,<0.87.0
```

Comment:
For more context, `fastapi` version `0.86.0` [pins to `starlette` version `0.20.4`]( https://github.com/tiangolo/fastapi/blob/0.86.0/pyproject.toml#L42 ), which was the last release before `0.87.0` (and so the latest version of `fastapi` allowed by this pin). Hence why the `starlette` installed has that CVE